Kernel/Arch/x86/common/Processor.cpp

```diff
@@ -663,7 +665,15 @@ UNMAP_AFTER_INIT void Processor::initialize(u32 cpu)
     if (cpu == 0) {
         VERIFY((FlatPtr(&s_clean_fpu_state) & 0xF) == 0);
         asm volatile("fninit");
-        if (has_feature(CPUFeature::FXSR))
+        // Initialize AVX state
+        if (has_feature(CPUFeature::XSAVE) && has_feature(CPUFeature::AVX))
```

Comment:
```suggestion
        if (has_feature(CPUFeature::XSAVE | CPUFeature::AVX))
```
We can check both features at once